modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/java/com/liferay/dynamic/data/lists/form/web/configuration/DDLFormWebConfiguration.java

```diff
@@ -44,4 +44,16 @@
 	)
 	public String defaultDisplayView();
 
+	@Meta.AD(
+		deflt = "show-warning", name = "show-csv-warning",
```

Comment:
Let's use the following labels instead:

CSV Export:
- Enabled
- Enabled (show warning)  [Default]
- Disabled